src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -391,18 +392,34 @@ p5.RendererGL.prototype._drawImmediateFill = function() {
   // WebGL 1 doesn't support the QUADS and QUAD_STRIP modes, so we
   // need to convert them to a supported format. In `vertex()`, we reformat
   // the input data into the formats specified below.
-  if (this.immediateMode.shapeMode === constants.QUADS) {
-    this.immediateMode.shapeMode = constants.TRIANGLES;
-  } else if (this.immediateMode.shapeMode === constants.QUAD_STRIP) {
-    this.immediateMode.shapeMode = constants.TRIANGLE_STRIP;
+  if (this.immediateMode.shapeMode === QUADS) {
```

Comment:
Why remove the .constants?

Reply:
Sorry, that must've been a mistake from when I left them out for testing. I fixed it in the latest commit.